docs/conf.py

```diff
@@ -54,102 +33,46 @@
 
 sphinx_tabs_disable_tab_closing = True
 
-# Control autodoc
-autoclass_content = "both"  # include init doc with class
+autoclass_content = "both"
 
-# Add any paths that contain templates here, relative to this directory.
```

Comment:
Why remove some of these comments?

Reply:
Ah I rebuilt this file completely and I think some of these comments are autogenerated from when this config file was built. Ill go back and parse out what's important.